assets/respec2.css

```diff
@@ -55,6 +55,16 @@ th code {
   color: inherit;
 }
 
+.orcid-logo {
+    display: inline-block;
+    width: 22px;
+    height: 16px;
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDowNzgwMTE3NDA3MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDo2REM1MzRDNUZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDo2REM1MzRDNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOjlFNTI0NkNBMDgyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjA3ODAxMTc0MDcyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+SwKtIQAAAjZJREFUeNpiZCASLDtnaQCkAoDYHogdcCg7AMQHgXhDlNHxC8SYy0iExQlAKh+IDRhIAyAHTAQ6ZAFZDgBarACk5uPxLbEAFCqJQIc8INoBQMsDoJYLMFAHfIA6YgNBB0CDfD4DbUAiepQwYvH5emQxMR4jBnFeQzD71MNpDJxs7JQ6IhA5JJiwxDkcfP/1E2y5jkQyGFMJzIfaheoAKsc5PiCA7FFGfPEOCgFyARFRBU4PLFBOPi5V8kKWKGkAXQwZ/PrzheH999sMr76cAzuegCNAdi5ggZZwOAsZWBpAdgCyGDbw6+8XhvvvtjKcfjiVgZn5Dy5lBiC7WaDFK1ng66/nQIu2ISKXU5VBEIi52SQZ1EXDwTno5MMWcKjgAAEs0LKdTAe8wJo1ZfjtGMzla8COAdE7bsTjMsKeiQpFLQZ48vEQw77b2WA2yBGg0MABHJholddAwQ6LHkVhL5zqmGiZ4Z98OAQPhQFxACg3EAI0dQAbM8/AOkBGwA6eHvA54AAtLAfFu6IQJPHdf7sNZ2OFCdqGo67PgeWAk+pUuO9vvl6JS+lBUEEEqpvrybGIm02CwUw+C2tJCLMcVBLiARtYQK1XYJl8gYxGJ9gibHUCkXXBBZDdsNpwIq5m2MvP54HkXIJiZNSGE1GaZMBQ2I9eLNOwPXAA6HtHEIMFuYEAxOeRW0VUaP/hbCFjlAPQdnsiHZpkKH0ElIII2lpNpLHlGwZ3x2RQdM0GReeUXt1zgAADALON7V7SXPS8AAAAAElFTkSuQmCC');
```

Comment:
I think the original SVG was fine, do you have a reason to do PNG instead?

Reply:
Agree. We should use the svg here.

Reply:
Ah wait... I see why @markasoftware was against putting this in the CSS. @saschanaz, if you see below, Mark added a div where the SVG would normally go. I was confused and thought the actual ID was being shown to the reader - when in fact, it's only the little logo next to their name. 

So, Mark was correct: the SVG should be in the document itself.

Reply:
For what it's worth, I tried to put the `background-image` on the link directly. However, this requires setting a width and height on the link, but width and height have no effect on inline elements. Setting the `display` to `inline-block` completely messes up the alignment, hence the separate div.

I'll add the SVG back into the HTML. Should I add an eslint exception so I can stuff the whole SVG on one line, or format it as before?

Reply:
Formatted it as before. Sorry for that additional work.

Reply:
Done!